test/end2end_test.go

```diff
@@ -6930,14 +6947,14 @@ func (s *httpServer) start(t *testing.T, lis net.Listener) {
 		}
 		writer.Flush() // necessary since client is expecting preface before declaring connection fully setup.
 		var sid uint32
-		// Loop until conn is closed and framer returns io.EOF
+		// Loop until conn is closed and framer returns possible Stop error messages
```

Comment:
Nit: replace `possible Stop error messages` with something more readable and terminate with a period.